cxx-sensors/src/tools/generate_cppcheck_resources.cmd

```diff
@@ -0,0 +1,14 @@
+@ECHO OFF
+
+SET SCRIPT_DIR=%~dp0
+SET CPPCHECK_DIR=C:\Program Files\Cppcheck
+SET PYTHON_DIR=C:\Perforce\Helix-QAC-2019.2\components\python-2.7.10
+
+SET CPPCHECK_LIBRARY_ARGS=--library=avr.cfg --library=boost.cfg --library=bsd.cfg --library=cppunit.cfg --library=embedded_sql.cfg --library=gnu.cfg --library=googletest.cfg --library=gtk.cfg --library=libcerror.cfg --library=microsoft_sal.cfg --library=motif.cfg --library=nspr.cfg --library=opengl.cfg --library=posix.cfg --library=python.cfg --library=qt.cfg --library=ruby.cfg --library=sdl.cfg
```

Comment:
as far as I remember the list of libraries was dynamic in the bash script; can you do the same on windows (in order new libraries are automatically added to the rules?)

Reply:
@ivangalkin yes, batch must be improved. Also download, unzip, ...